onnx/cpp2py_export.cc

```diff
@@ -324,6 +327,8 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
   // Submodule `shape_inference`
   auto shape_inference = onnx_cpp2py_export.def_submodule("shape_inference");
   shape_inference.doc() = "Shape Inference submodule";
+  py::register_exception<InferenceError>(shape_inference, "InferenceError");
```

Comment:
Is this intended to cover both shape-inference errors as well as type-errors? Wondering why module is called "shape inference", while the exception is plain "InferenceError".

Reply:
yes both module and exception class are intended to handle both type and shape inference.... I am simply registering the exception here...